scripts/release_scripts/run_macaron.sh

```diff
@@ -140,6 +147,64 @@ function assert_path_exists() {
     fi
 }
 
+# Create a directory if it does not exist.
+# Arguments:
+#   $1: The directory to create.
+function create_dir_if_not_exists() {
+    dir=$1
+    if [ ! -d "$dir" ]; then
+        mkdir --parents "$dir"
+    fi
+}
+
+# Add a directory to the list of volume mounts stored in the ``mounts`` global variable.
+#
```

Comment:
I think it would be good to rename the function to indicate that it can potentially create a directory.

In addition, please add a note here that this function can have side effect, i.e., if the mounted directory does not exist, it will create it.